docs/shu_bmrc.md

```diff
@@ -1,13 +1,88 @@
 # nf-core/configs: Sheffield Hallam University BMRC Cluster Configuration
 
-## Using the Institutional Configuration Profile
+This document provides guidelines for using nf-core pipelines on Sheffield Hallam University's BMRC High-Performance Computing (HPC) cluster. The custom configuration file for this cluster enables optimised resource usage and workflow compatibility within the BMRC HPC environment, facilitating efficient execution of nf-core workflows.
 
-To use [`shu_bmrc.config`](../conf/shu_bmrc.config), run nextflow with an nf-core pipeline using `-profile shu_bmrc` (note the single hyphen).
+---
 
-This will download and launch [`shu_bmrc.config`](../conf/shu_bmrc.config) which has been pre-configured with a setup suitable for the BMRC cluster and will automatically load the appropriate pipeline-specific configuration file.
+## Table of Contents
 
-## Running nf-core Pipelines on the BMRC Cluster
+1. [Introduction](#introduction)
+2. [Requirements](#requirements)
+3. [Configuration Details](#configuration-details)
+4. [Usage](#usage)
+5. [Troubleshooting](#troubleshooting)
+6. [Support and Contact](#support-and-contact)
 
-A detailed guide to setting up Nextflow and nf-core and running pipelines on the BMRC Cluster is available here: [Running Nextflow with nf-core Pipelines on SHU BMRC Cluster](https://github.com/lquayle88/nfcore_on_shu_bmrc)
+---
 
-If you have any questions or issues not addressed by this guide, please contact: [l.quayle@shu.ac.uk](mailto:l.quayle@shu.ac.uk)
+## Introduction
+
+This configuration file is specifically designed for running nf-core workflows on the BMRC HPC cluster at **Sheffield Hallam University**. The configuration integrates optimal resource parameters and scheduling policies to ensure efficient job execution on the cluster, aligning with internal HPC policies and specifications.
+
+The cluster configuration:
+
+- **Location**: BMRC HPC Cluster at Sheffield Hallam University
+- **Contact**: Dr Lewis A Quayle ([l.quayle@shu.ac.uk](mailto:l.quayle@shu.ac.uk))
+- **Documentation**: [BMRC HPC Documentation](https://bmrc-hpc-documentation.readthedocs.io/en/latest/)
+
+## Requirements
+
+To use this configuration, you must have:
+
+- **Access to BMRC HPC**: Ensure your user account is enabled for HPC access at Sheffield Hallam University. The **GlobalProtect VPN** is required for remote access. For setup instructions, refer to [SHU VPN Guide](https://www.shu.ac.uk/digital-skills/programs-and-applications/virtual-private-network-vpn).
+- **Nextflow**: Version 20.10.0 or later is recommended for optimal compatibility.
```

Comment:
That's extremely old, are you still using DSL1 pipelines? If so I would recommend to use 22.10.6 as the minimum version (the last DSL1 supporting version of Nextflow)

Reply:
Thanks James. This was carryover from an old guide I wrote (another win for copy-paste). I will update accordingly.